MCprep_addon/world_tools.py

```diff
@@ -170,8 +170,13 @@ def convert_mtl(filepath):
 	mtl = filepath.rsplit(".", 1)[0] + '.mtl'
 	lines = None
 	copied_file = None
-	with open(mtl, 'r') as mtl_file:
-		lines = mtl_file.readlines()
+
+	try:
+		with open(mtl, 'r') as mtl_file:
```

Comment:
I'd also argue we may as well do the check for which files exist a couple lines before.

If we know blanket-wise it converts spaces to underscore, let's just add it in now:

```python
mtl_underscore = os.path.join(
    os.path.dirname(mtl_underscore),
    os.path.basename(mtl_underscore).replace(" ", "_")
if not os.path.isfile(mtl) and os.path.isfile(mtl_underscore):
    mtl = mtl_underscore
```

.. but would be good to sanity check it too, of course.

Reply:
I've went ahead and implemented an existence check in the beginning of the function, but with pathlib:
```py
mtl = Path(filepath.rsplit(".", 1)[0] + '.mtl')
if not mtl.exists():
	mtl_underscores = Path(mtl.parent.absolute()) / mtl.name.replace(" ", "_")
	if mtl_underscores.exists():
		mtl = mtl_underscores
	else:
		return False
```

Sidenote: Pathlib seems to be a lot easier and more readable when doing file operations, we should use it more